fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/properties/CookieConfigProperties.java

```diff
@@ -0,0 +1,21 @@
+package de.adorsys.opba.fintech.impl.properties;
+
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.lang.Nullable;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
+@ConfigurationProperties(prefix = "server.servlet.session.cookie")
```

Comment:
server.servlet.session.cookie prefix doesn't seem to have anything specific with servlet, suggest to move it to 
auth.cookie